### ISSUE TITLE
Shot Generator is not working

### ISSUE THREAD

**Issue by javegavictor** [wonderunit/storyboarder#2505]
When I try to use the shot generator the screen goes grey, and i can't do anything
![Screenshot 2024-01-14 222127](https://github.com/wonderunit/storyboarder/assets/131863062/661a4139-784b-417c-9b58-f0772f8b6e43)
![Screenshot 2024-01-14 222150](https://github.com/wonderunit/storyboarder/assets/131863062/bc96fb38-90ec-421d-99ab-d8dadf2d5910)

**Comment by cleargulf** [comment]
Yes. I tried setting windows compatibility mode to Windows 8 and ran as administrator. A lot of the missing features of shot generator appeared. Posing, female figures, etc.  Worth a try. Magic.:)
